addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -51,38 +53,33 @@ function animateClone($cart, $elem, offsetTop, offsetLeft) {
         $(this).removeClass("o_red_highlight").dequeue();
     });
     return new Promise(function (resolve, reject) {
-        var $imgtodrag = $elem.find('img').eq(0);
-        if ($imgtodrag.length) {
```

Comment:
Put that guard at the top of the function since the `$imgtodrag` is now given to the function